sceptre/cli/dump.py

```diff
@@ -48,8 +52,70 @@ def dump_config(ctx, path):
 
     output_format = "json" if context.output_format == "json" else "yaml"
 
+    def process_config(config):
+        stack_name = list(config.keys())[0]
+        file_path = Path(f".dump/{stack_name}/config.yaml")
+        write(config[stack_name], output_format, no_colour=True, file_path=file_path)
+
     if len(output) == 1:
-        write(output[0][stack.external_name], output_format)
+        process_config(output[0])
     else:
         for config in output:
-            write(config, output_format)
+            process_config(config)
+
+
+@dump_group.command(name="template")
+@click.argument("path")
+@click.option(
+    "-n",
+    "--no-placeholders",
+    is_flag=True,
+    help="If True, no placeholder values will be supplied for resolvers that cannot be resolved.",
+)
```

Comment:
I think there ought to be an option, like `--only` that will exclusively output the template at the indicated path and not include any other configs passed along. Probably the same for `dump config` as well.

Reply:
Yes possibly, but could we do that in a separate change later?

Reply:
That's a pretty simple change to make, IMO. But yes, it could be done later.